README.md

```diff
@@ -93,10 +93,10 @@ x86, x86-64, LoongArch64, 32-bit & 64-bit Little Endian MIPS, 32-bit PowerPC (Bi
 support WebAssembly.
 For more information, see [the supported `ring` target platforms][ring-target-platforms].
 
-By providing a custom instance of the [`crate::crypto::CryptoProvider`] struct, you
+By providing a custom instance of the crypto::CryptoProvider struct, you
```

Comment:
Instead of removing the links what about specifying an explicit target? E.g. like the `[ring-target-platforms]` instance. That seems to work in both Rustdocs and the README.